flow/project.py

```diff
@@ -470,14 +615,15 @@ def __eq__(self, other):
 
 
 class BaseFlowOperation(object):
-    """A BaseFlowOperation represents a data space operation, operating on any job.
+    """A BaseFlowOperation represents a data space operation, operating on any
+    job or aggregate of jobs.
 
     Every BaseFlowOperation is associated with a specific command.
 
     Pre-requirements (pre) and post-conditions (post) can be used to
     trigger an operation only when certain conditions are met. Conditions are unary
-    callables, which expect an instance of job as their first and only positional
-    argument and return either True or False.
+    callables, which expect an instance of job or aggregate of jobs as their first
+    and only positional argument and return either True or False.
```

Comment:
This isn't true though given the other code, we expect the signature to accept multiple positional arguments for aggregates correct?